Unquote.netstandard.sln

```diff
@@ -24,81 +23,84 @@ Global
 		Release|x64 = Release|x64
 		Release|x86 = Release|x86
 	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{9CE24E83-361D-4064-A5EA-1CADD5EA1E71}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{9CE24E83-361D-4064-A5EA-1CADD5EA1E71}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{9CE24E83-361D-4064-A5EA-1CADD5EA1E71}.Debug|x64.ActiveCfg = Debug|x64
-		{9CE24E83-361D-4064-A5EA-1CADD5EA1E71}.Debug|x64.Build.0 = Debug|x64
-		{9CE24E83-361D-4064-A5EA-1CADD5EA1E71}.Debug|x86.ActiveCfg = Debug|x86
-		{9CE24E83-361D-4064-A5EA-1CADD5EA1E71}.Debug|x86.Build.0 = Debug|x86
+		{9CE24E83-361D-4064-A5EA-1CADD5EA1E71}.Debug|x64.ActiveCfg = Debug|Any CPU
```

Comment:
you didnt mean these, I bet ;)

Reply:
Ok I amended the commit to remote the useless changes on sln file 👍